### ISSUE TITLE
possible threading crash

### ISSUE THREAD

**Issue by PiggyDr** [TeamPeril/Sculk-Hoard#71]
iirc these occured in sculk commander dev environment, upon or shortly after loading world. unsure if they're my fault or not but i figured i'd open an issue here just in case. i was using forge 47.3.0 sculk horde 0.9.11 so it's possible that it's already been fixed. will update to 0.9.14 and let you know if anything happens
[latest (16).log](https://github.com/user-attachments/files/17006497/latest.16.log)
[latest (17).log](https://github.com/user-attachments/files/17006498/latest.17.log)

**Comment by PiggyDr** [comment]
update: [still happening](https://github.com/user-attachments/files/17006658/latest.log)
it's common enough that i'm pretty sure this is a sculk commander issue

**Comment by goreae** [comment]
I am having the same issue. This instance worked just fine before I started updatin mods. No origins or sculk commander. At this point I can't enter my world for more than a second without getting an "accessing legacy random source from multiple threads" error. Tried with forge and neoforge, same result. Tried disabling other mods I added, same result.

On a hunch, I downgraded the version to 0.9.10. Works fine. 0.9.11 and up all crash within seconds of loading into the world. It might have been introduced with the performance tweaks that happened in that version. 
In any case, here's my crash report: [crash-2024-09-16_11.55.41-server.txt](https://github.com/user-attachments/files/17017637/crash-2024-09-16_11.55.41-server.txt)
Here's the latest.log: [latest.log](https://github.com/user-attachments/files/17017656/latest.log)
